### ISSUE TITLE
 There is no output result after running demo

### ISSUE THREAD

**Issue by forevergogi** [xingyizhou/CenterNet#482]
When I debug the demo.py, I find that, there is no output result in the expected directory. It seems that the code `output = self.model(images)[-1]` exits with no information. You can see it in the picture below.I run it in a RTX 2080Ti GPU server and the env is Cuda 10.0 + Pytorch 1.20+ Python 3.6. Actually, I have asked you the questions of not loading fc.weight and fc.bias before. It seems the error is about the model loading problem. Can you help to solve this problem? Many thanks!
![tmp](https://user-images.githubusercontent.com/8411980/69396463-3e615400-0d1d-11ea-8a1e-ccb9737975f0.jpg)

**Comment by forevergogi** [comment]
![tmp](https://user-images.githubusercontent.com/8411980/69396493-5f29a980-0d1d-11ea-9c9b-aa7b8e222794.jpg)

**Comment by xingyizhou** [comment]
I didn't understand. Do you mean the program exists without printing out anything? I will have no idea about this. I heard there are some problems using RTX, you can search the issues to see if there are something helpful.

**Comment by forevergogi** [comment]
> I didn't understand. Do you mean the program exists without printing out anything? I will have no idea about this. I heard there are some problems using RTX, you can search the issues to see if there are something helpful.

Yes, the program exits without printing any running error info. I think it exits when testing the image.

**Comment by forevergogi** [comment]
> I didn't understand. Do you mean the program exists without printing out anything? I will have no idea about this. I heard there are some problems using RTX, you can search the issues to see if there are something helpful.

I found why the program goes dump error, because the gcc version is 4.8 in CentOS 7, but we need to user gcc 4.9 above to compile the DCN module. After I git clone the DCNv2 code and re-compile the code with gcc 5.5, it goes well.
